app/src/sharedTest/java/org/oppia/android/app/topic/revision/TopicRevisionFragmentTest.kt

```diff
@@ -180,21 +214,25 @@ class TopicRevisionFragmentTest {
 
   private fun createTopicActivityIntent(internalProfileId: Int, topicId: String): Intent {
     return TopicActivity.createTopicActivityIntent(
-      ApplicationProvider.getApplicationContext(), internalProfileId, topicId
+      context = ApplicationProvider.getApplicationContext(),
+      internalProfileId = internalProfileId,
+      topicId = topicId
     )
   }
 
   private fun launchTopicActivityIntent(
     internalProfileId: Int,
     topicId: String
   ): ActivityScenario<TopicActivity> {
-    return launch(createTopicActivityIntent(internalProfileId, topicId))
+    return launch(
+      createTopicActivityIntent(internalProfileId = internalProfileId, topicId = topicId)
+    )
   }
 
   private fun clickRevisionTab() {
     onView(
       allOf(
-        withText(TopicTab.getTabForPosition(3).name),
+        withText(TopicTab.getTabForPosition(position = 3).name),
```

Comment:
Ditto here.

Reply:
@prayutsu We do require a named argument here as simply passing `3` sometime not makes the code readable as we cannot simply understand what is the number meaning we are passing as an argument. So, we do require this named argument here.

Reply:
@anandwana001 Thanks for clarifying. Can you also verify this below?